### DISCUSSION TITLE
User config plugin ?

### DISCUSSION THREAD

**Discussion by silvyn** [kookma/TW-Tamasha#10] [Ideas]
Hi Mohammad,

Do you plan to add customizable user options?

I'm thinking for example of a general option not to display the slide numbers (the progress bar is already very nice and would be enough for me, where then I don't like the round style ^^').

**Comment by kookma** [comment]
@silvyn - that is a good suggestion. it is not difficult to implement such configuration. I appreciate to let me know what other features you like to be configurable. 

To keep Tamasha simple one suggestion is to use themes for these features e.g a theme shows page number and progress bar another not

**Reply by silvyn** [reply]
Yep, theme config may be enough.

**Reply by kookma** [reply]
I will look at theme to see how I implement this!
